src/styles/components/index.scss

```diff
@@ -0,0 +1,33 @@
+.home {
+  position: relative;
+}
+
+.home-top {
+  padding: 13rem 0.5rem 0 8rem;
+  max-width: 47%;
+
+  h1 {
+    font-size: 2.5rem;
+  }
+
+  &__top-description {
+    font-size: 1.1rem;
+  }
+
+  &__top-image {
+    position: absolute;
+    right: 1rem;
+    top: 1rem;
+  }
+}
+
+@media (max-width: 1258px) {
+  .home-top {
+    max-width: 100%;
```

Comment:
If you give the home container a display: flex and make a container for the inner items that is display:grid and set up to the grid-template-rows you would like then you may not need a media query.

I got my [band card codepen](https://codepen.io/tisidro/pen/OJvqrMx) to work in this way w/out any media queries (thanks to Zastrow's help!)

Reply:
Great work on this Bennett!  🐰 The last item is just an idea--a media query works just as well. I'm not sure what the best max-width would be but I'm sure you can get good ideas posting on apprentice or in the dev channel! 😊

Reply:
I've added a breakpoint variable in the globals.scss, which is where we can put all our breakpoint variables. Although I used an scss varriable, because the normal ones were not working for some reason.